### PR TITLE
Be sure to only push the newly created tag

### PR DESCRIPTION
**Pull request by SgtCoDFish**
Prompted by [this](https://kubernetes.slack.com/archives/CDEQJ0Q8M/p1655373470751709)

### CODE REVIEW

File: content/docs/contributing/release-process.md [hunk 1]
```diff
@@ -215,7 +215,8 @@ page if a step is missing or if it is outdated.
 
        ```bash
        git tag -s -m"v1.8.0-beta.0" v1.8.0-beta.0
-       git push --tags
+       # be sure to push the named tag explicitly; you don't want to push any other local tags!
+       git push v1.8.0-beta.0
```
Comment (munnerz) [inline content/docs/contributing/release-process.md:219]: Should the remote being pushed to be included as well? `git push upstream v1.8.0-beta.0` :)

Comment (SgtCoDFish) [reply content/docs/contributing/release-process.md]: thanks, fixed!

**Comment by maelvls** [comment]
Nice! This single fix will prevent unexpected tags being pushed

**Pull request merged**